OpenRA.Game/Translation.cs

```diff
@@ -92,9 +92,6 @@ public string GetString(string key, IDictionary<string, object> arguments = null
 
 		public bool TryGetString(string key, out string value, IDictionary<string, object> arguments = null)
 		{
-			if (key == null)
-				throw new ArgumentNullException(nameof(key));
-
```

Comment:
What exactly is the issue in this PR in regards to null strings? `MediaGlobal` has anti-null checks as well as `AddTransientLine`.